ofApp.cpp

```diff
@@ -1,387 +0,0 @@
-/*
-READ ME
-Please note that when looking through app it is much easier to debug / test when you can accurately know the answer, therefore I have left in
-all of my code which displays the answers on screen but commented it out. These code lines have a "password" next to them.
-Simply hold CTRL and hit 'F' and type in:
-//YzxNkP99T9WYb8
-Removing the comments from these lines will help :)
-*/
-#include "ofApp.h" // included so my objects and classes and variables work correctly
-#include <stdlib.h> // included to i dont have to write STD every time i want to print something 
-
-#include <time.h> // included to give me access to random numbers
-
-ofApp leftObj; // create 2 objects here
-ofApp rightObj;
-
-void ofApp::setup(){ // setup code only runs once 
-    /*
-        API documentation can be found at: https://restcountries.com/
-    */
-    json.open("https://restcountries.com/v3.1/all"); // opens the api and populates it with the information
```

Comment:
if you need to refresh the data you could refactor this into a separate funciton